app/config/paths.js

```diff
@@ -15,10 +14,14 @@ let confDir = homeDir;
 const devDir = resolve(__dirname, '../..');
 const devConfig = resolve(devDir, conf);
 const defaultConfig = resolve(__dirname, defaultConf);
-const pathLegacy = resolve(homeDir, legacyConf);
 
 const icon = resolve(__dirname, '../static/icon.png');
 
+const keymapPath = resolve(__dirname, '../keymaps');
+const darwinKeys = resolve(keymapPath, 'darwin.json');
```

Comment:
True enough for this particular path, but that would be the reasoning nonetheless.